test/import/module_path/foo.mod.bash

```diff
@@ -0,0 +1,3 @@
+foo::bar() {
```

Comment:
Would it be useful at all to have `importmod` add the `foo::` automatically? I tried it out for functions and it was pretty straightforward, at the expense of using subshells and sourcing the module multiple times. Function calls still need to use the fully-qualified names since they wouldn't be executed until after the module prefix is added to the function declarations.

Reply:
I was thinking about something similar but in reverse, i.e. importmod could dynamically strip `foo::` from each function if the user requests it.

Curious how you're picturing the implementation. Would you have `importmod` automatically generate wrapper functions for each qualified name?

Reply:
I just sourced the script in a subshell, used `declare -f $name` to print the definition, and then threw `foo::` on the front and `eval`ed it. Pretty dirty but at least I didn't have to write a parser.